lib/pbench/server/database/alembic/env.py

```diff
@@ -1,62 +1,28 @@
-"""Alembic Migration Driver
-
-This file was auto generated by `alembic init alembic` but was manually altered
-to suit the needs of the Pbench server. Re-running `alembic init alembic` will
-overwrite these changes!
-
-This Python script runs whenever the alembic migration tool is invoked in
-/opt/pbench-server/lib/server/database; it contains instructions to configure
-and generate a SQLAlchemy engine, procure a connection from that engine along
-with a transaction, and then invoke the migration engine, using the connection
-as a source of database connectivity.
-
-This requires access to Pbench library modules and the Pbench server config file
-and thus must be run with
-
-    export PYTHONPATH=/opt/pbench-server/lib:${PYTHONPATH}
-    export _PBENCH_SERVER_CONFIG=/opt/pbench-server/lib/config/pbench-server.cfg
-
-Examples:
-
-   alembic upgrade head     # upgrade database to the latest
-   alembic downgrade base   # downgrade to the original tracked state
-"""
-import logging
-import sys
+from logging.config import fileConfig
 
 from alembic import context
-from sqlalchemy import create_engine
+from sqlalchemy import engine_from_config, pool
 
-from pbench.server.api import get_server_config
 from pbench.server.database.database import Database
 
-# this is the Alembic Config object, which provides
-# access to the values within the .ini file in use.
+# This is the Alembic Config object, which provides access to the values within
+# the .ini file in use.
 config = context.config
 
-# Add syslog handler to send logs to journald
-log = logging.getLogger("alembic")
-handler = logging.handlers.SysLogHandler("/dev/log")
-log.addHandler(handler)
-
-# add your model's MetaData object here for 'autogenerate' support:
+# Interpret the config file for Python logging and setup the loggers.
+fileConfig(config.config_file_name)
 
+# Add your model's MetaData object here for 'autogenerate' support:
+#     from myapp import mymodel
+#     target_metadata = mymodel.Base.metadata
```

Comment:
Are these lines generated?  If not, I would remove lines 16-17, since they are now represented by lines 6 and 18, respectively.  (And, I would remove that first `y`.  😉)

Reply:
Seems like we should keep the auto-generated code.  I re-wrapped it a bit since we are re-formatting the file, but removing the comment seems like a loss.

Reply:
I disagree:  line 15 is instructions which you performed already when you edited the file, so we don't need to retain it (at least, not as written -- you could modify it to make it more applicable to what you've done...), and lines 16-17 are sample code which have been superseded by actual code (at lines 6 and 18), so they should be removed, as well.

Reply:
The question is, "is it really worth re-submitting a change to remove or change this further?"  The comments don't hurt, and they are the original comments just reformatted, which I believe is a good thing.  I'd like to escape and just submit this PR so we can land the check which is more import.  The comment styles don't seem to be material to our code base.

Reply:
The situation is that Alembic generates a file for us which we then edit to provide context which Alembic doesn't have access to.  Those edits should be kept small so that the material change relative to the generated file is clear.  We really shouldn't be making superficial or cosmetic changes, like reformatting generated comments -- that just generates noise for future reviewers.

Now, whether it is better to leave the above comment in place or remove it is a smaller question.  Leaving it in place will make it disappear from the diff, which is good.  Removing it will also highlight the fact that we replaced it with actual code, which is also good.

So, I can be persuaded to go either way on removing the comment, but I really don't think it should be touched in any other way -- that's counterproductive.

However, speaking of counterproductive, I guess I'll just resolve this conversation, now.